demos/evp/aesgcm.c

```diff
@@ -42,15 +43,15 @@ static const unsigned char gcm_ct[] = {
     0xb9, 0xf2, 0x17, 0x36
 };
 
-static const unsigned char gcm_tag[] = {
+static unsigned char gcm_tag[] = {
```

Comment:
Resolves this warning:
> aesgcm.c:115:69: warning: passing argument 4 of 'EVP_CIPHER_CTX_ctrl' discards 'const' qualifier from pointer target type [-Wdiscarded-qualifiers]
     EVP_CIPHER_CTX_ctrl(ctx, EVP_CTRL_GCM_SET_TAG, sizeof(gcm_tag), gcm_tag);